app/containers/App/selectors.js

```diff
@@ -10,22 +10,13 @@ const selectGlobal = state => state.global || initialState;
 const selectRouter = state => state.router;
 
 const makeSelectCurrentUser = () =>
```

Comment:
As per https://github.com/react-boilerplate/react-boilerplate/issues/2806#issuecomment-554855162, do we want to change some of these selector factories to regular selectors?

Reply:
Yes it's in the to do list for #2812. The changes you see here are just because prettier wanted them.